### DISCUSSION TITLE
Is it possible to have multiple selection menus?

### DISCUSSION THREAD

**Discussion by dituff** [discord-jda/JDA#2121] [Questions and Help]
Is it possible to have multiple selection menus with sequential choice and common response ? Now after selecting the first menu, he immediately tries to get the answer.

**Comment by ShuraBlack** [comment]
Interactions fail after 3 seconds of no response, so you are forced to reply with the bot (withint 3 seconds) or use deferReply, to set the bot to "Thinking". You can read about this on https://github.com/DV8FromTheWorld/JDA/wiki/Interactions (under Responding to Slash Commands)
